cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -136,12 +136,28 @@ function($, Backbone, _, gettext, BaseView, XBlockViewUtils, MoveXBlockUtils, Ht
             }
         },
 
+        isValidCategory: function(sourceParentType, targetParentType, targetHasChildren) {
+            var basicBlockTypes = ['course', 'chapter', 'sequential', 'vertical'];
+            // Support move component under content experiment. Treat source parent components as a verticals.
```

Comment:
Treat source parent components as vertical to support move child components under content experiment and other similar xblocks.